proto/ibc/applications/transfer/v1/tx.proto

```diff
@@ -38,6 +38,8 @@ message MsgTransfer {
   // Timeout timestamp in absolute nanoseconds since unix epoch.
   // The timeout is disabled when set to 0.
   uint64 timeout_timestamp = 7 [(gogoproto.moretags) = "yaml:\"timeout_timestamp\""];
+  // optional metadata
+  bytes metadata = 8;
```

Comment:
We should update [the docs](https://github.com/cosmos/ibc-go/blob/main/docs/apps/transfer/messages.md#msgtransfer) with this addition.